portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,8 +41,19 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String quote = quotes.get((int) (Math.random() * quotes.size()));
-    response.setContentType("text/html;");
-    response.getWriter().println(quote);
+    // String quote = quotes.get((int) (Math.random() * quotes.size()));
+    response.setContentType("application/JSON;");
+    response.getWriter().println(buildJSON());
+  }
+
+  private String buildJSON() {
+    String json = "{";
+    json += "\"Freshman_Summer\": ";
+    json += "\"European Bioinformatics Institute\"";
+    json += ", ";
```

Comment:
nit: You could alternatively use str.format here to construct this by creating a map of <year,  projectname> etc to make this slightly easier, it is sad that java doesn't have nice support for multiline strings. this is an optional suggestion.

Reply:
Yeah, that would've been a better choice... Thanks!